<launch>
	<rosparam ns="object_detection" command="load" file="$(find object_detection)/params/object_color.yaml"></rosparam>
	<node pkg="object_detection" name="object_color_range" type="object_color_range.py" args="object_color" output="screen">
		<param name="segmentation_topic" type='str' value='/semantic_segmentation/compressed'/>
	</node>
</launch>